src/containers/User/SettingsProfileAvatar.styles.js

```diff
@@ -0,0 +1,6 @@
+import { StyleSheet } from 'react-native'
+
+export const styles = StyleSheet.create({
+  container: {
```

Comment:
Do you want this as a placeholder? Maybe we can just add it when there are styles?